app/components/post_list/post_list_base.js

```diff
@@ -72,6 +80,10 @@ export default class PostListBase extends PureComponent {
             this.handleDeepLink(this.props.deepLinkURL);
             this.props.actions.setDeepLinkURL('');
         }
+
+        this.lastPostIndex = this.getLastPostIndex(this.props.postIds);
```

Comment:
Why getting this value on willMount and didMount?

Reply:
It's not necessary.  That was part of my initial code . Thanks for catching.